sdk/instance/start.go

```diff
@@ -31,6 +31,7 @@ func (i *Instance) start(inst *instance.Instance) (serviceIDs []string, err erro
 	}
 	_, port, _ := xnet.SplitHostPort(conf.Server.Address)
 	endpoint := conf.Name + ":" + strconv.Itoa(port)
+	env := xos.EnvMapToSlice(xos.EnvMergeMaps(xos.EnvSliceToMap(srv.Configuration.Env), xos.EnvSliceToMap(inst.Env)))
```

Comment:
```suggestion
	env := append(srv.Configuration.Env, inst.Env...)
```

or just use EnvMergeSlices

Reply:
https://github.com/mesg-foundation/engine/pull/1060#discussion_r295254438